zio-http/src/main/scala/zhttp/service/Client.scala

```diff
@@ -156,7 +156,7 @@ object Client {
   ) extends HeaderExtension[ClientRequest] { self =>
 
     def getBodyAsString: Option[String] = data match {
```

Comment:
We can add `getBodyAsCharSequence` also. It's alright if you want to do it in a different PR :)

Reply:
done